### PR TITLE
[IMP] Inventory: rewrite DHL integration

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR fully rewrites the doc on integrating DHL with the Odoo Inventory app.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @kekiss,

This PR is ready for the first round of review.

Thanks,
John

**Review by kekiss — APPROVED** [review]
No notes! Great job! :star2:

**Comment by hojo-odoo** [comment]
Hey @ksc-odoo,

This PR is ready for final review.

Thanks,
John

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 1]
```diff
@@ -1,22 +1,139 @@
...
+
+Once the developer portal account has been confirmed, log in to the portal using the username and
+password. Click on the user avatar in the top-right corner of the screen to open the user dashboard.
+
+On the dashboard, click on the :guilabel:`Apps` tab, and select an app. In the
+:guilabel:`Credentials` section, click the :guilabel:`Show key` button below the :guilabel:`API Key`
+and :guilabel:`API Secret` fields.
+
+Return to the shipping method form in the Odoo database. Enter the :guilabel:`DHL API Key` in the
+:guilabel:`SiteID` field, and the :guilabel:`API Secret` in the :guilabel:`DHL Password` field.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:109]:
```suggestion
Return to the shipping method form in the Odoo database. Enter the *DHL API Key* in the
:guilabel:`DHL SiteID` field, and the *API Secret* in the :guilabel:`DHL Password` field.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 2]
```diff
@@ -1,22 +1,139 @@
...
+
+Shipping details
+~~~~~~~~~~~~~~~~
+
+The rest of the fields in the :guilabel:`DHL Configuration` tab are used to configure the shipping
+method itself:
+
+- :guilabel:`Region`: the region in which the shipping method is used.
+- :guilabel:`DHL Product`: the shipping service purchased from DHL (ex. Express Worldwide).
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:118]:
```suggestion
- :guilabel:`DHL Product`: the shipping service purchased from DHL (e.g. Express Worldwide).
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 3]
```diff
@@ -1,22 +1,139 @@
...
+- :guilabel:`DHL Package Type`: the type of DHL package used for delivery (ex. DHL Box).
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:119]:
```suggestion
- :guilabel:`DHL Package Type`: the type of DHL package used for delivery (e.g. DHL Box).
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 4]
```diff
@@ -1,22 +1,139 @@
...
+- :guilabel:`Package Weight Unit`: the unit of measure used to display package weight.
+- :guilabel:`Package Dimension Unit`: the unit of measure used to display package size.
+- :guilabel:`Label Format`: the file format used to generate shipping labels.
+- :guilabel:`Label Template`: the paper size used to print shipping labels.
+
+.. important::
+   Before selecting service options for a shipping method, make sure that those services are
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:126]:
```suggestion
   Before selecting service options for a shipping method, make sure those services are
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 5]
```diff
@@ -1,22 +1,139 @@
...
+   actually available for the DHL account. Available services depend on the contract negotiated with
+   DHL.
+
+Options
+~~~~~~~
+
+A couple of additional settings are available in the :guilabel:`Options` section at the bottom of
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:133]:
```suggestion
Additional settings are available in the :guilabel:`Options` section at the bottom of
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 6]
```diff
@@ -1,22 +1,139 @@
...
+the :guilabel:`DHL Configuration` tab:
+
+- :guilabel:`Generate Return Label`: enable this option to automatically generate a return label
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:136]:
```suggestion
- :guilabel:`Generate Return Label`: Enable this option to automatically generate a return label
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 7]
```diff
@@ -1,22 +1,139 @@
...
+  after validating a delivery order.
+- :guilabel:`Dutiable Material`: enable this option if the shipping method is liable to customs or
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:138]:
```suggestion
- :guilabel:`Dutiable Material`: Enable this option if the shipping method is liable to customs or
```

**Review by ksc-odoo — APPROVED** [review]
hey @hojo-odoo -- just finished my Final Review of this PR. Great work. Just have a handful of minor suggestions that require your attention. Once you implement the necessary changes, you can feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 8]
```diff
@@ -1,22 +1,139 @@
...
+.. image:: dhl_credentials/dhl-form.png
+   :align: center
+   :alt: The form for a DHL shipping method.
+
+General information
+-------------------
+
+Begin configuring the shipping method by entering its title in the :guilabel:`Shipping Method`
+field.
+
```
[the remark this answers is not in the document]
Comment (hojo-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:60]: That one is general to all shippers, so it's not covered in this doc.

**Comment by hojo-odoo** [comment]
Hey @samueljlieber,

This PR is ready for tech review.

Thanks,
John

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst [hunk 9]
```diff
@@ -1,22 +1,137 @@
...
 
-Getting SiteID and password for countries other than United States (UK and Rest of the world)
-=============================================================================================
+Once the connector has been enabled, a :icon:`oi-arrow-right` :guilabel:`DHL Shipping Methods` link
+appears below. Click the link to open a page showing all shipping methods configured for DHL.
 
-You should contact DHL account manager and request integration for XML Express API. The presales should provide you live credentials.
+.. image:: dhl_credentials/dsm.png
+   :align: center
+   :alt: The DHL Shipping Methods button on the Settings page of the Inventory app.
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dhl_credentials.rst:30]: Only suggestion is to remove this image, as it is not entirely necessary IMO :)
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Great work on this doc @hojo-odoo, this is a significant improvement over the current version of this doc. I only have one small suggestion to remove an image, but please feel free to keep it if you feel differently!

Thank you for your work!
.....
@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**